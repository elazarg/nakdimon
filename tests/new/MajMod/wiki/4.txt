הקרבין פּוֹתֵחַ בְּ-1942 עַל יְדֵי חֶבְרַת הַנֶּשֶׁק וינצ'סטר, בעיצומה שֶׁל מִלְחֶמֶת הָעוֹלָם הַשְּׁנִיָּיה, כרובה חצי-אוטומטי מְקוּצָר וְקַל שֶׁנּוֹעַד ללוחמי קו-אחורי (כְּגוֹן מַפְעִילֵי אַרְטִילֶרְיָה) ותומכי לְחִימָה כְּנֶשֶׁק הֲגָנָה עַצְמִית, בַּמָּקוֹם אֶקְדָּח הַשֵּׁירוּת קוֹלֵט M1911. הֱיוֹתוֹ רוֹבֶה קַל וּמְדוּיָּק, חצי-אוטומטי ומוזן מחסנית הָפְכוּ אוֹתוֹ לַנֶּשֶׁק פּוֹפּוּלָרִי גַּם בְּקֶרֶב לוֹחֲמֵי צנחנים וְכוֹחוֹת קומנדו. הוּא יוֹצֵר בִּלְמַעְלָה משישה מִילְיוֹן יְחִידוֹת, ושרת בְּכוֹחוֹת המזוינים שֶׁל אַרְצוֹת הַבְּרִית כִּשְׁלוֹשָׁה עֲשׂוֹרִים, וּמְאוּחָר יוֹתֵר שָׂרַת גַּם בְּקֶרֶב כּוֹחוֹת צָבָא ומשטרה שֶׁל בַּעֲלוֹת הַבְּרִית שֶׁל אָמֵרִיקָה, בֵּינֵיהֶן יִשְׂרָאֵל. הקרבין שָׂרַת גַּם בְּצַהַ"ל בִּשְׁנוֹתָיו הַמּוּקְדָּמוֹת ושרת כְּאַרְבָּעָה עֲשׂוֹרִים בְּמִשְׁטֶרֶת יִשְׂרָאֵל ובמשמר הָאֶזְרָחִי.