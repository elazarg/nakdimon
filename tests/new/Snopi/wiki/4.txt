הקרבין פּוֹתֵחַ ב-1942 עַל יְדֵי חֲבֵרַת הַנֶּשֶׁק וינצ'סטר, בעיצומה שֶׁל מִלְחֶמֶת הָעוֹלָם הַשְּׁנִיָּיה, כְּרוֹבֶה חִצֵּי-אוֹטוֹמָטִי מְקוּצָּר וְקַל שֶׁנּוֹעַד ללוחמי קַו-אֲחוֹרִי (כְּגוֹן מפעילי ארטילריה) וְתוֹמְכַי לְחֵימָה כְּנֶשֶׁק הֵגֵנָּה עצמית, בִּמְקוֹם אֶקְדָּח הַשֵּׁירוּת קוֹלֵט M1911.
הֱיוֹתוֹ רוֹבֶה קַל ומדויק, חִצֵּי-אוֹטוֹמָטִי ומוזן מַחְסָנִית הָפְכוּ אוֹתוֹ לַנֶּשֶׁק פּוֹפּוּלָרִי גַּם בַּקְּרָב לוחמי צַנְחָנִים וְכוֹחוֹת קוֹמַנְדּוֹ.
הוּא יוֹצֵר בַּלְּמַעְלָה מִשִּׁישָּׁה מִילְיוֹן יְחִידוֹת, וְשַׁרְתָּ בַּכּוֹחוֹת המזוינים שֶׁל אַרְצוֹת הַבְּרִית כִּשְׁלוֹשָׁה עָשׂוֹרִים, וּמְאוּחָר יוֹתֵר שַׁרְתָּ גַּם בִּקְרָב כּוֹחוֹת צָבָא וּמִשְׁטָרָה שֶׁל בַּעֲלוּת הַבְּרִית שֶׁל אָמֶרִיקָה, בֵּינֵיהֶן יִשְׂרָאֵל.
הקרבין שַׁרְתָּ גַּם בְּצַהַ"ל בִּשְׁנוֹתָיו הַמּוּקְדָּמוֹת וְשָׁרָת כארבעה עָשׂוֹרִים במשטרת יִשְׂרָאֵל וּבַמִּשְׁמָר הָאֶזְרָחִי.