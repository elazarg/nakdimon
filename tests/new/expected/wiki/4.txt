הַקָּרַבִּין פּוּתַּח בְּ-1942 עַל יְדֵי חֶבְרַת הַנֶּשֶׁק וִינְצֶ'סְטֵר, בְּעִיצּוּמָהּ שֶׁל מִלְחֶמֶת הָעוֹלָם הַשְּׁנִיָּיה, כְּרוֹבֶה חֲצִי-אוֹטוֹמָטִי מְקוּצָּר וְקַל שֶׁנּוֹעַד לְלוֹחֲמֵי קַו-אֲחוֹרִי (כְּגוֹן מַפְעִילֵי אַרְטִילֶרְיָה) וְתוֹמְכֵי לְחִימָה כְּנֶשֶׁק הֲגַנָּה עַצְמִית, בִּמְקוֹם אֶקְדָּח הַשֵּׁירוּת קוֹלְט M1911.
הֱיוֹתוֹ רוֹבֶה קַל וּמְדוּיָּק, חֲצִי-אוֹטוֹמָטִי וּמוּזָן מַחְסָנִית הָפְכוּ אוֹתוֹ לְנֶשֶׁק פּוֹפּוּלָרִי גַּם בְּקֶרֶב לוֹחֲמֵי צַנְחָנִים וְכוֹחוֹת קוֹמַנְדּוֹ.
הוּא יוּצַּר בִּלְמַעְלָה מִשִּׁישָּׁה מִילְיוֹן יְחִידוֹת, וְשֵׁרֵת בַּכּוֹחוֹת הַמְּזוּיָּנִים שֶׁל אַרְצוֹת הַבְּרִית כִּשְׁלוֹשָׁה עֲשׂוֹרִים, וּמְאוּחָר יוֹתֵר שֵׁרֵת גַּם בְּקֶרֶב כּוֹחוֹת צָבָא וּמִשְׁטָרָה שֶׁל בַּעֲלוֹת הַבְּרִית שֶׁל אָמֵרִיקָה, בֵּינֵיהֶן יִשְׂרָאֵל.
הַקָּרַבִּין שֵׁרֵת גַּם בְּצַהַ"ל בִּשְׁנוֹתָיו הַמּוּקְדָּמוֹת וְשֵׁרֵת כְּאַרְבָּעָה עֲשׂוֹרִים בְּמִשְׁטֶרֶת יִשְׂרָאֵל וּבַמִּשְׁמָר הָאֶזְרָחִי.